package.json

```diff
@@ -65,10 +65,10 @@
     "webpack-dev-server": "^2.9.3"
   },
   "scripts": {
-    "unit": "jest --testPathPattern=.*.test.js$",
+    "unit": "jest --testPathPattern=.*.test.js$ --coverage",
```

Comment:
with python tests we generally ran the tests and the coverage separately. Is there a reason why you're combining it into one line here?

Reply:
@akegan two feed two birds with one seed - but I can separate them -- would you like me to do that?

Reply:
My only concern is that it might be hard to distinguish on semaphore if the tests are failing due to tests failing or due to coverage not passing. would that be an issue here?

Reply:
This just reports the coverage that gets passed to code climate, it doesn't keep it from passing or failing. Code climate will let us know in the PR if it doesn't pass. e.g. this PR shows that the new changes don't pass the Diff Test Coverage:
![screen shot 2019-02-27 at 1 35 20 pm](https://user-images.githubusercontent.com/5904673/53524575-a20f6700-3a94-11e9-831c-67fb47063fc3.png)